core/src/main/resources/jenkins/install/pluginSetupWizard.properties

```diff
@@ -56,3 +58,17 @@ installWizard_pluginInstallFailure_message=Some plugins failed to install proper
 with the failed plugins
 installWizard_continue=Continue
 installWizard_retry=Retry
+installWizard_upgradePanel_title=Upgrade
+installWizard_upgradePanel_banner=Welcome to Jenkins 2!
+installWizard_upgradePanel_message=Jenkins 2 includes some great new features we think you'll love to use. Get your instance updated to the latest \
```

Comment:
@rtyler More a question from a non native speaker: didn't we say using the full form _you will_ instead of _you'll_ would be preferable in the UI texts? 

Possibly I saw some fixes along those lines in the recent PRs @orrc did about improving many helps and so on.